src/wp_rsa_sig.c

```diff
@@ -432,14 +436,14 @@ static int wp_rsa_signverify_init(wp_RsaSigCtx* ctx, wp_Rsa* rsa,
     if (ok) {
         ctx->saltLen = RSA_PSS_SALTLEN_AUTO;
 
-        if (wp_rsa_get_type(rsa) == RSA_FLAG_TYPE_RSA) {
+        if (wp_rsa_get_type(ctx->rsa) == RSA_FLAG_TYPE_RSA) {
```

Comment:
No need for this. ctx->rsa == rsa at this point and on.

Reply:
As I mentioned in the commit message, this is just a defensive coding. Currently `ctx->rsa == rsa`, but there may be a future change above that inadvertently violates this. Additionally, this doesn't use too much more code space.